common/clickhouse-srv/Cargo.toml

```diff
@@ -0,0 +1,69 @@
+[package]
+name = "common-clickhouse-srv"
+version = "0.3.2"
+authors = ["sundyli <543950155@qq.com>"]
+edition = "2018"
```

Comment:
I think it should be migrated to the 2021 edition

Reply:
FYI https://blog.rust-lang.org/2021/07/21/Rust-2021-public-testing.html

Reply:
ok !

Reply:
Done.